.github/workflows/python_test.yml

```diff
@@ -0,0 +1,40 @@
+name: Python application test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install -e .
+        pip3 install absl-py pytest Pillow
```

Comment:
should add these dependencies into setup.py instead.

Reply:
Done